client/stylesheets/_toCat.scss

```diff
@@ -797,6 +797,10 @@ input[type="search"]::-webkit-search-results-decoration { display: none; }
     margin-right: 2.5px;
 }
 
+#datepicker {
+    color: black !important;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces